src/Shared/LayoutRenderers/AspNetRequestClientCertificateLayoutRenderer.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Text;
+using NLog.LayoutRenderers;
+#if ASP_NET_CORE
+using Microsoft.AspNetCore.Http;
+#else
+using System.Security.Cryptography.X509Certificates;
+using System.Web;
+#endif
+
+namespace NLog.Web.LayoutRenderers
+{
+    /// <summary>
+    /// ASP.NET Client Certificate of the Connection
+    /// </summary>
+    /// <remarks>
+    /// ${aspnet-request-client certificate}
+    /// </remarks>
+    [LayoutRenderer("aspnet-request-client certificate")]
```

Comment:
Could the space be replaced with a dash. Ex. "aspnet-request-client-certificate"

Reply:
Sorry missed a dash